actionview/test/template/date_helper_test.rb

```diff
@@ -399,7 +399,7 @@ def test_select_month_with_abbv
   end
 
   def test_select_month_with_custom_names
-    month_names = %w(nil Januar Februar Marts April Maj Juni Juli August September Oktober November December)
+    month_names = %w(nil January February March April May June July August September October November December)
```

Comment:
We want the previous version since we want to use custom month names, not the regular names.